website/src/theme/ArticleChecklist/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styles from './styles.module.css'
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+const expDays = 2000;
+
+function updateValues(id, state) {
+    let dt = new Date();
+    dt.setTime(dt.getTime() + (expDays * 24 * 60 * 60 * 1000));
+    const cookie = `${id}=${state}; expires=${dt.toUTCString()}; path=/`
+    console.log("Updated cookie "+cookie)
+    document.cookie = cookie
+}
+
+function getCookie(name) {
+    const nameEQ = name + "=";
+    const ca = document.cookie.split(';');
+    for (let i = 0; i < ca.length; i++) {
+      const c = ca[i];
+      while (c.charAt(0) == ' ') c = c.substring(1, c.length);
+      if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length, c.length);
+    }
+    return null;
+  }
+
+export function isChecked(id) {
+    const v = getCookie(id);
+    console.log("COokie: "+id+" "+v)
```

Comment:
Will these `console.log`s show up in the browser console for our users? If so, they should be removed.

Reply:
no, the checklist is going to be activated in another PR - it's disabled in this one. 

Sadly, I needed to merge the maturity checklist in here to get some other changes that were needed.